public/plugin_init.rb

```diff
@@ -1,7 +1,9 @@
 ## Register our custom page action
-AppConfig[:pui_page_custom_actions] << {
-  'record_type' => ['archival_object', 'accession'],
-  'erb_partial' => 'aeon/aeon_request_action'
-}
+unless AppConfig.has_key?(:pui_page_custom_actions)
```

Comment:
What is the purpose of including this? To me, it looks like this is excluding this plugin from the PUI when it already contains a custom action.

Reply:
Yeah, this was clumsy.

The goal was to allow a different list of record types from config. Turns out config wins here anyway, so it was unnecessary, as well as having the unpleasant side effect you noted.

I've backed this one out - thanks.

Reply:
Actually this was double clumsy and a case of forgetting what happened a few days ago ;)

It is necessary to avoid having two entries in :pui_page_custom_actions because it will cause two request buttons to appear.

I've just pushed a commit that allows specifying the supported record types from config. This feels ok to me. Let me know what you think.

Ideally AppConfig wouldn't be getting set from plugin_init, but I get why it has been done that way.